### PR TITLE
[TW-1222] Doc sharing performance test reports

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

File: src/pages/docs/collections/testing-api-performance.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ From the **Summary** tab, you can take the following actions:
 * Select **Stop** to stop the performance test.
 * Select **Run Again** to configure and start the performance test again.
 * Select **All Runs** to view a list of all past performance runs for the collection.
+* Select the share icon <img alt="Share icon" src="https://assets.postman.com/postman-docs/icon-share.jpg#icon" width="16px"> to get a link to the performance report. Select **Copy Link** to copy the link  your clipboard. You can share the link with your teammates so they can view the results of the performance test.
```
Comment (petewitcher-postman) [inline src/pages/docs/collections/testing-api-performance.md:108]:
```suggestion
* Select the share icon <img alt="Share icon" src="https://assets.postman.com/postman-docs/icon-share.jpg#icon" width="16px"> to get a link to the performance report. Select **Copy Link** to copy the link to your clipboard. You can share the link with your teammates so they can view the results of the performance test.
```

**Review by petewitcher-postman — APPROVED** [review]
LGTM! Found a typo.

File: src/pages/docs/collections/testing-api-performance.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/collections/testing-api-performance.md:108]: Yikes, thanks Pete!

**Review by postfrzero — APPROVED** [review]

**Review by nerminamiller-postman — APPROVED** [review]
LGTM!

**Review by malvikach — APPROVED** [review]

**Pull request merged**